Resources/public/templates/universaldiscovery/explorerlevel.hbt

```diff
@@ -0,0 +1,25 @@
+<a class="ez-ud-finder-explorerlevel-anchor">.</a>
+<div class="ez-ud-finder-explorerlevel-content ez-asynchronousview">
+    <div class="ez-ud-finder-explorerlevel-loading ez-font-icon"></div>
+    {{#if loadingError}}
+        <p class="ez-asynchronousview-error ez-font-icon">
+            <button class="ez-asynchronousview-retry ez-button ez-font-icon pure-button"></button>
+        </p>
+    {{else}}
+        <ul class="ez-explorer-level-list">
+            {{#each items}}
+                <li class="ez-explorer-level-list-item ez-font-icon
+                    {{#if location.childCount}} is-parent-location{{/if}}
+                    {{#if selectedLocationId}} is-selected{{/if}}
+                    {{#if ../ownSelectedItem}} has-selected-item{{/if}}"
+                >
+                    <a class="ez-explorer-level-item ez-font-icon{{#if selectedLocationId}} is-selected{{/if}}"
```

Comment:
given you set add an ellipsis when that's too long, you should add a title attribute with the full name